src/routes/routes.js

```diff
@@ -49,6 +54,20 @@ router.post('/login_parent', login_parent.post);
 router.post('/register_parent', register_parent.post);
 router.post('/add_child', add_child.post);
 
+router.get('/schedule_pickup', (req, res) => {
+  parent_children_and_da(req.session.parent_id).then((queryRes) => {
```

Comment:
yess, this is the proper way to handle a promise 👍 (related to my comments in #29 )